whois-api/src/main/java/net/ripe/db/whois/api/httpserver/FilteredPasswordSlf4jRequestLogWriter.java

```diff
@@ -1,17 +1,17 @@
 package net.ripe.db.whois.api.httpserver;
 
+import net.ripe.db.whois.common.conversion.PasswordFilter;
 import org.eclipse.jetty.server.Slf4jRequestLogWriter;
 
 import javax.annotation.Nonnull;
 import java.io.IOException;
-import java.util.regex.Pattern;
 
-public class FilteredPasswordSlf4jRequestLogWriter extends Slf4jRequestLogWriter {
 
-    private static final Pattern PASSWORD_PATTERN = Pattern.compile("(?<=(?i)(password=))([^&]*)");
+public class FilteredPasswordSlf4jRequestLogWriter extends Slf4jRequestLogWriter {
```

Comment:
Rename back to `FilteredSlf4RequestLogWriter` as it's not just filtering passwords

Reply:
done in 44045027c